srcd/dashboards/gitbase/overview.json

```diff
@@ -518,6 +521,7 @@
                 "description": null,
                 "fetch_values_predicate": null,
                 "filter_select_enabled": false,
+                "is_sqllab_view": false,
```

Comment:
why is this one the only false?

Reply:
because one of the charts uses default pre-registered table. Pre-registred tables should be visible.